packages/workbench/src/app/modules/login/login.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="login-form" id="wb-login">
+  <div class="card">
```

Comment:
We should be using primeng in the new workbench, but I won't insist for doing it in this MR